content/english/anti-vegan/pests.md

```diff
@@ -0,0 +1,150 @@
+
+[Biosecurity NZ](http://web.archive.org/web/20191007214252/https://www.mpi.govt.nz/protection-and-response/long-term-pest-management/wild-rabbits/) lists rabbits as pests because they ‘compete with livestock, cause extensive damage to farming land, cause public nuisance… 
+
+So, yes, they are annoying to (some) humans but they are not a ‘pest’. The reasons provided by Biosecurity NZ fail under Leopold’s criteria. If wild rabbits are a pest other more substantial, non-speciesist, reasons must be provided.
+
+### Are possums a pest?
+
+Possums occur in high numbers in NZ, breed fast and face very few predators. Without doubt possums have a major impact on New Zealands native ecosystems.
+
+[Department of Conservation NZ](http://web.archive.org/save/https://www.doc.govt.nz/nature/pests-and-threats/animal-pests/possums/) lists possums as pests, as, “Leaves are the main part of their diet, but possums are opportunistic omnivores. They eat buds, flowers, fruit/berries and nectar, which means they compete with native birds and reptiles for food sources. The growth and life-cycle of a tree or plant is significantly affected when all parts of it are eaten. Possums also have ‘favourites’ such as rātā or kamahi trees, leading to an even greater impact on these species.  …Possums were [also]filmed eating the eggs and chicks of kōkako …  They eat invertebrates, including weta, and are significant predators of New Zealand land snails such as Powelliphanta. They often occupy holes in tree trunks for their nests which would otherwise be used by nesting birds such as kākāriki and saddlebacks. …Researchers using nest-cameras have witnessed … kea nests invaded by stoats and possums in South Westland. ”
+
+Here it would seem possums can be defined as a ‘pest species’ that directly impacts the ecosystem, without any reference to human specific interests.
+
+### To recap
+
+Vegans may still object to the term 'pest'. A species that is out-of-place in the eco-system is, in most cases, due to past meddling by humans and not a fault of the species. 
+
+Vegans can classify a species as problematic by reference to Leopold's non-speciesist definition. However, once identified, any action taken must *as far as is possible and practicable* be neither exploitive nor unnecessarily cruel.
+
+### So where to from here? 
+
+While the ideals of veganism are always placed first, individual vegans will differ on what, if any, actions ought to be taken. This can range from 'do nothing' (*all species have a right to life*) to 'controlling possums' (applying the golden rule of *acting to bring about the least harm*)
+
+This is discussed in more depth in the appendix.
+
+### Final thoughts
+
+Identifying a species as problematic does not commit a vegan to act in any specific manner. However, that vegans can offer a non-speciesist definition of 'pest' and can argue for a specific approach in dealing with them demonstrates that vegans can address complex environmental concerns.
```

Comment:
This paragraph in particular reads quite repetitiously after 'So where to from here?" Can we combine these two sections?

Reply:
Does a bit... so yup